Composer/packages/client/src/pages/design/SideBar.tsx

```diff
@@ -107,7 +107,7 @@ const SideBar: React.FC<SideBarProps> = React.memo(({ projectId }) => {
     if (link.botError) {
       setBrokenSkillInfo(link);
     }
-    const { skillId, dialogId, trigger } = link;
+    const { skillId, dialogId, trigger, projectId } = link;
```

Comment:
The diff makes it hard to see how this value is being used.

Reply:
Can you point me to where its being used?

Reply:
It's used immediately below in the calls like `navTo(skillId ?? projectId, dialogId);`. Before this change, the value of projectId was obtained from the outside scope - a prop that was handed to the SideBar component - and that was defined out in DesignPage.tsx also as `skillId ?? projectId`, so every dialog link the side bar ended up navigating to whichever projectId was currently "active" instead of their own internal projectId.

Reply:
Can we consider renaming some variables for clarity? It's very difficult to know what `projectId` is when used in the various places.